packages/services/usage/src/usage.ts

```diff
@@ -140,10 +168,11 @@ export function createUsage(config: {
       };
     },
     // settings recommended by Azure EventHub https://docs.microsoft.com/en-us/azure/event-hubs/apache-kafka-configurations
-    requestTimeout: 60_000, //
+    requestTimeout: 30_000,
```

Comment:
recommendation is 30-60s.
Our average duration is 190ms, so 30s is more than enough.